tests/cases/dynamicRunner.ts

```diff
@@ -0,0 +1,20 @@
+/// <reference path="../../src/compiler/types.ts" />

```

Comment:
This also wouldn't quite work post-modules since these files won't declare namespaces that can be assumed to exist, they'll each have individual exports that contribute to the greater API. Maybe, they could be switched to import a the barrel module in the runner, though.

Reply:
Yep I thought later these would be replaced by re-exports from imports so things remain working